### PR TITLE
Windows 7 support

### PR DESCRIPTION
**Pull request by rihardsgravis**
# Description

Mock Realm storage and revert to local storage

## Type of change

- Bug fix (a non-breaking change which fixes an issue)
- Enhancement (a non-breaking change which adds functionality)

# How Has This Been Tested?

Tested manually on Window 7

# Checklist:

- [x] My code follows the style guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/desktop/native/preload/Electron.js [hunk 1]
```diff
@@ -194,15 +197,30 @@ const Electron = {
...
+        Object.keys(keys).forEach((key) => {
+            this.removeStorage(key);
+        });
+        setTimeout(() => {
+            storageLock = false;
+        }, 3000);
+    },
+
+    /**
+     * Remove all epracated redux-persist local storage items
```
Comment (rajivshah3) [inline src/desktop/native/preload/Electron.js]:
```suggestion
     * Remove all deprecated redux-persist local storage items
```

File: src/desktop/native/preload/Electron.js [hunk 2]
```diff
@@ -194,15 +197,30 @@ const Electron = {
...
+     */
+    clearOldStorage() {
         const keys = electronSettings.getAll();
-        Object.keys(keys).forEach((key) => this.removeStorage(key));
+        Object.keys(keys).forEach((key) => key.indexOf('reduxPersist') === 0 && this.removeStorage(key));
     },
 
     /**
-     * Get all local storage items
+     * Get all depracated redux-persist local storage items
```
Comment (rajivshah3) [inline src/desktop/native/preload/Electron.js]:
```suggestion
     * Get all deprecated redux-persist local storage items
```

File: src/desktop/native/Realm.js [hunk 3]
```diff
@@ -1,7 +1,14 @@
-import Realm from 'realm';
-
-// Don't report Realm JS analytics data
-// @see https://github.com/realm/realm-js/blob/master/lib/submit-analytics.js#L28
-process.env.REALM_DISABLE_ANALYTICS = 'true';
+/**
+ * Realm database mock for Windows 7 compataibillity
```
Comment (rajivshah3) [inline src/desktop/native/Realm.js]:
```suggestion
 * Realm database mock for Windows 7 compatibility
```

**Comment by rihardsgravis** [comment]
Closing this PR as it is not meant to be merged at any time.

**Pull request closed without merging**